package/cfg/k3s-cis-1.8-permissive/node.yaml

```diff
@@ -0,0 +1,457 @@
+        remediation: |
+          Run the below command (based on the file location on your system) on the each worker node.
+          For example,
+          chmod 600 $kubeletkubeconfig
+        scored: true
+
+      - id: 4.1.6
+        text: "Ensure that the --kubeconfig kubelet.conf file ownership is set to root:root (Automated)"
+        audit: 'stat -c %U:%G /var/lib/rancher/k3s/agent/kubelet.kubeconfig'
+        tests:
+          test_items:
+            - flag: root:root
+        remediation: |
+          Run the below command (based on the file location on your system) on the each worker node.
+          For example,
+          chown root:root $kubeletkubeconfig
+        scored: true
+
+      - id: 4.1.7
+        text: "Ensure that the certificate authorities file permissions are set to 600 or more restrictive (Manual)"
+        audit: "stat -c %a /var/lib/rancher/k3s/server/tls/server-ca.crt"
+        tests:
+          test_items:
+            - flag: "permissions"
+              compare:
+                op: bitmask
+                value: "600"
+        remediation: |
+          Run the following command to modify the file permissions of the
+          --client-ca-file chmod 600 <filename>
+        scored: false
+
+      - id: 4.1.8
+        text: "Ensure that the client certificate authorities file ownership is set to root:root (Manual)"
+        audit: "stat -c %U:%G /var/lib/rancher/k3s/server/tls/client-ca.crt"
+        tests:
+          test_items:
+            - flag: root:root
+              compare:
+                op: eq
+                value: root:root
+        remediation: |
+          Run the following command to modify the ownership of the --client-ca-file.
+          chown root:root <filename>
+        scored: false
+
+      - id: 4.1.9
+        text: "Ensure that the kubelet --config configuration file has permissions set to 600 or more restrictive (Automated)"
+        type: "skip"
+        audit: '/bin/sh -c ''if test -e $kubeletconf; then stat -c permissions=%a $kubeletconf; fi'' '
+        tests:
+          test_items:
+            - flag: "permissions"
+              compare:
+                op: bitmask
+                value: "600"
+        remediation: |
+          Run the following command (using the config file location identified in the Audit step)
+          chmod 600 $kubeletconf
+        scored: true
+
+      - id: 4.1.10
+        text: "Ensure that the kubelet --config configuration file ownership is set to root:root (Automated)"
+        type: "skip"
+        audit: '/bin/sh -c ''if test -e $kubeletconf; then stat -c %U:%G $kubeletconf; fi'' '
+        tests:
+          test_items:
+            - flag: root:root
+        remediation: |
+          Run the following command (using the config file location identified in the Audit step)
+          chown root:root $kubeletconf
+          Not Applicable.
+          All configuration is passed in as arguments at container run time.
+        scored: true
```

Comment:
```suggestion
      - id: 4.1.9
        text: "Ensure that the kubelet --config configuration file has permissions set to 600 or more restrictive (Automated)"
        audit: '/bin/sh -c ''if test -e $kubeletconf; then stat -c permissions=%a $kubeletconf; fi'' '
        tests:
          test_items:
            - flag: "permissions"
              compare:
                op: bitmask
                value: "600"
        remediation: |
          Run the following command (using the config file location identified in the Audit step)
          chmod 600 $kubeletconf
        scored: true

      - id: 4.1.10
        text: "Ensure that the kubelet --config configuration file ownership is set to root:root (Automated)"
        audit: '/bin/sh -c ''if test -e $kubeletconf; then stat -c %U:%G $kubeletconf; fi'' '
        tests:
          test_items:
            - flag: root:root
        remediation: |
          Run the following command (using the config file location identified in the Audit step)
          chown root:root $kubeletconf
        scored: true
```
4.1.9 and 4.1.10 should not be skipped (notApplicable) for K3S either in permissive or hardened. Indeed when provisioning K3s (default), a kubelet.kubeconfig is present and permissions are set to root:root and 600 by default. So we expect this check to be automated (scored: true) and to PASS.

With the current global config.yaml, we need also to override kubeletconf in k3s-cis-1.8-permissive/config.yaml, related to https://github.com/rancher/cis-operator/issues/211.

Suggested changes to be also likely applied:

- K3s - CIS-1.8 hardened/permissive
- K3s - CIS-1.7 hardened/permissive - to confirm if K3s 1.25's kubeletconfig is provisioned in root:root + 600.
- RKE2 - CIS-1.8 hardened/permissive - to be confirmed
- RKE2 - CIS-1.7 hardened/permissive - to be confirmed


Here's the output of the modified (remove skip + add kubeletconf path) k3s-cis-1.8-permissive scan (4.1.9/4.1.10) against a `v1.26.11+k3s2` :

```
root@k3s-standalone-k3s-standalone:~# ./kube-bench --config-dir security-scan/package/cfg/ --config security-scan/package/cfg/config.yaml --benchmark=k3s-cis-1.8-permissive -c 4.1.9,4.1.10 --nosummary
[INFO] 1 Control Plane Security Configuration

[INFO] 2 Etcd Node Configuration

[INFO] 3 Control Plane Configuration

[INFO] 4 Worker Node Security Configuration
[INFO] 4.1 Worker Node Configuration Files
[PASS] 4.1.9 Ensure that the kubelet --config configuration file has permissions set to 600 or more restrictive (Automated)
[PASS] 4.1.10 Ensure that the kubelet --config configuration file ownership is set to root:root (Automated)

[INFO] 5 Kubernetes Policies

== Summary total ==
2 checks PASS
0 checks FAIL
0 checks WARN
0 checks INFO
```

@rayandas I can create either have a PR or commit with the changes.